spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -3029,4 +3042,11 @@ module Crystal
       assert_macro %({{ (x.conds << "a"; x.conds.size) }}), "2", {x: node}
     end
   end
+
+  describe "int128 support" do
+    it "" do
+      node = When.new([2.int32, 3.int32] of ASTNode, 4.int32)
+      assert_macro %({{ (x.conds << "a"; x.conds.size) }}), "2", {x: node}
+    end
+  end
```

Comment:
What does this example do with 128-bit integers?

Reply:
Oops, looks like I just copied this spec:
https://github.com/crystal-lang/crystal/blob/baf5c3b3045c8401c4364a6e7f351d314b1e1fb5/spec/compiler/macro/macro_methods_spec.cr#L3027-L3030